caravel/migrations/versions/1226819ee0e3_fix_wrong_constraint_on_table_columns.py

```diff
@@ -0,0 +1,50 @@
+"""Fix wrong constraint on table columns
+
+Revision ID: 1226819ee0e3
+Revises: 956a063c52b3
+Create Date: 2016-05-27 15:03:32.980343
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '1226819ee0e3'
+down_revision = '956a063c52b3'
+
+from alembic import op
+import sqlalchemy as sa
+
+naming_convention = {
+    "fk":
+    "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
+}
+
+def find_constraint_name(upgrade = True):
```

Comment:
On top of this should we probabily generalize this function so can be reused in other migrations?